### PR TITLE
Ensure *all* builder components are built & published in script

### PR DESCRIPTION
**Pull request by reset**
Signed-off-by: Jamie Winsor <jamie@vialstudios.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: support/ci/publish-builder.sh [hunk 1]
```diff
@@ -10,7 +10,12 @@
 set -eu
 src_root=$(cd "$(dirname "$0")/../../"; pwd)
 
-for crate in `find components/builder-*/habitat -maxdepth 0 -type d`; do
+if [ -z ${HAB_AUTH_TOKEN+x} ]; then
+  echo "HAB_AUTH_TOKEN var is unset"
+  exit 1
+fi
+
```
Comment (mwrock) [inline support/ci/publish-builder.sh]: NICE!

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/182164612).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/182164612).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**